_includes/neurips/2019/about.neurips.2019 - pr.html

```diff
@@ -81,6 +81,9 @@ <h3>Platinum</h3>
                   <a target="_blank" href="https://www.facebook.com"><img width=500
```

Comment:
This file was changed in a previous pull request so there are merge conflicts blocking acceptance of these changes. Please pull the latest version of the master branch, remove merge conflicts, and re-push changes for this page.